### DISCUSSION TITLE
solving with other Numerical Methods (MOL)

### DISCUSSION THREAD

**Discussion by MohammdReza5106** [idaholab/moose#24914] [Q&A General]
Hi there,

While working on my thesis, I came across this amazing framework. I'm wondering if it's possible to solve PDE-type problems with other numerical solutions. As an MSc. student in Welding Engineering, I'm particularly interested in using this framework to solve problems related to SMAW moving heat sources and temperature fields. However, in order to meet the requirements of my thesis proposal, I need to use the Method of Lines approach instead. Can you tell me if this is possible with the framework?

**Comment by GiudGiud** [comment]
Hello

I suspect the method of lines is similar to the method of characteristics (MOC) (but with the characteristic being a line).
If so, then it can be done using the ray tracing module. We have a neutron transport MOC solver, so it can be done, we just can't share it.

Guillaume

**Reply by MohammdReza5106** [reply]
Dear Guillaume
i really appreciate your guide but as i have a limited time to do my work , i wonder that isn't there any written code in these method
that i learn and modified them to adjust my case
thank you.

**Reply by GiudGiud** [reply]
There's a few examples in the module
We also have this example doing MPM
https://github.com/loganharbour/moose/tree/rt_mpm
